<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="35078" started-at="2016-04-13T20:53:40Z" finished-at="2016-04-13T20:54:15Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="35078" started-at="2016-04-13T20:53:40Z" finished-at="2016-04-13T20:54:15Z">
      <class name="paqueteejer1.Ejer1">
        <test-method status="PASS" signature="beforeMethod(org.testng.ITestContext)[pri:0, instance:paqueteejer1.Ejer1@193948d]" name="beforeMethod" is-config="true" duration-ms="9699" started-at="2016-04-13T17:53:40Z" finished-at="2016-04-13T17:53:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@188af83]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="ejercicio1()[pri:0, instance:paqueteejer1.Ejer1@193948d]" name="ejercicio1" duration-ms="25131" started-at="2016-04-13T17:53:49Z" description="Ejercicio1" finished-at="2016-04-13T17:54:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[si se encuentra la palabra noticia en el comentario expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: si se encuentra la palabra noticia en el comentario expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at paqueteejer1.Ejer1.ejercicio1(Ejer1.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:75)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:46)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ejercicio1 -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestContext)[pri:0, instance:paqueteejer1.Ejer1@193948d]" name="afterMethod" is-config="true" duration-ms="215" started-at="2016-04-13T17:54:14Z" finished-at="2016-04-13T17:54:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@188af83]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- paqueteejer1.Ejer1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
